src/Employer/Employer.Web/Controllers/ApplicationReviewsController.cs

```diff
@@ -26,20 +27,26 @@ public ApplicationReviewsController(IApplicationReviewsOrchestrator orchestrator
 
         [FeatureGate(FeatureNames.MultipleApplicationsManagement)]
         [HttpGet("unsuccessful", Name = RouteNames.ApplicationReviewsToUnsuccessful_Get)]
-        public async Task<IActionResult> ApplicationReviewsToUnsuccessful(VacancyRouteModel rm)
+        public async Task<IActionResult> ApplicationReviewsToUnsuccessful(VacancyRouteModel rm, [FromQuery] string sortColumn, [FromQuery] string sortOrder)
         {
-            var viewModel = await _orchestrator.GetApplicationReviewsToUnsuccessfulViewModelAsync(rm);
+            Enum.TryParse<SortOrder>(sortOrder, out var outputSortOrder);
+            Enum.TryParse<SortColumn>(sortColumn, out var outputSortColumn);
+
+            var viewModel = await _orchestrator.GetApplicationReviewsToUnsuccessfulViewModelAsync(rm, outputSortColumn, outputSortOrder);
 
             return View(viewModel);
         }
 
         [FeatureGate(FeatureNames.MultipleApplicationsManagement)]
         [HttpPost("unsuccessful", Name = RouteNames.ApplicationReviewsToUnsuccessful_Post)]
-        public async Task<IActionResult> ApplicationReviewsToUnsuccessfulAsync(ApplicationReviewsToUnsuccessfulRouteModel rm)
+        public async Task<IActionResult> ApplicationReviewsToUnsuccessfulAsync(ApplicationReviewsToUnsuccessfulRouteModel rm, [FromQuery] string sortColumn, [FromQuery] string sortOrder)
```

Comment:
is there a reason why this isnt on the model being posted? for the sortColumn and sortOrder?

Reply:
The reason is that if the user does not select any checkboxes, then the server-side sorting is "preserved", it basically goes to the db again to get the same applications sorted in the same order. Not sure if there is a way around it? Open to suggestions. Maybe the tech debt ticket EC-114 would be a solution, removing the modelstate validations?

Reply:
No reason. I moved it inside the posted model